VolleyManagement.UI/spa/environments/environment.integrated.ts

```diff
@@ -0,0 +1,11 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=prod` then `environment.prod.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
+  production: true,
+  integrated: true,
```

Comment:
"I don't know if it possible, maybe for someone who really good in webpack and angular cli configuration, because you want config which will use files with variable environment in one case and with variable vouIntegration in other. I can simply rename field inside environment from "integrated" to  "vouIntegration" if you really think that it is needed))"

I was talking about variable name primarily - but I'm interested what was you talking about?)

Reply:
1. you placed comment under "environment", which is really variable in JS terms
2. If you mean field "integrated" inside object assigned to environment const, it is not a problem, just let me know exactly